packages/login/test/login-overlay.test.js

```diff
@@ -159,17 +163,20 @@ describe('title and description', () => {
     expect(descriptionElement.textContent).to.be.equal(overlay.description);
   });
 
-  it('should update title and description when property updated', () => {
+  it('should update title and description when property updated', async () => {
     overlay.title = 'The newest title';
     overlay.description = 'The newest description';
+    await nextUpdate(overlay);
 
     expect(headerElement.textContent).to.be.equal(overlay.title);
     expect(descriptionElement.textContent).to.be.equal(overlay.description);
   });
 
-  it('should update title and description when i18n.header updated', () => {
+  it('should update title and description when i18n.header updated', async () => {
     const i18n = { ...overlay.i18n, header: { title: 'The newest title', description: 'The newest description' } };
     overlay.i18n = i18n;
+    await nextUpdate(overlay);
+    await nextUpdate(overlay.$.vaadinLoginOverlayWrapper);
```

Comment:
We could consider using [`getUpdateComplete`](https://lit.dev/docs/components/lifecycle/#getUpdateComplete) in the element instead. 
However, I don't think it makes sense to implement it in such a simple use case.